docs/zh_CN/zookeeper/zookeeper.md

```diff
@@ -22,7 +22,7 @@ Cluster-configuration.
 
 * Manual build
 
-refer to [dockerfile](../../../src/zookeeper/build/zookeeper.dockerfile).
+refer to [dockerfile](../../../src/zookeeper/build/zookeeper.yarn.dockerfile).
```

Comment:
better to use `service_type` in config, otherwise, you need to rename files again if there're three types of modes and the service is used by two of them in the future